### PR TITLE
RIA-15547 update asertion panel view

### PR DESCRIPTION
**Pull request by AlexanderYeremeyev**
https://smartbear.atlassian.net/browse/RIA-15547

### CODE REVIEW

File: src/main/java/com/smartbear/mqttsupport/teststeps/panels/PublishTestStepPanel.java [hunk 1]
```diff
@@ -81,23 +81,23 @@ public PublishTestStepPanel(PublishTestStep modelItem) {
...
-
-
         add(inspectorPanel.getComponent());
         setPreferredSize(new Dimension(500, 300));
 
     }
 
     private JComponent buildMainPanel() {
         JPanel root = new JPanel(new MigLayout("wrap", "0[grow,fill]0", "0[]0[grow,fill]0"));
+        root.setBorder(BorderFactory.createMatteBorder(
```
Comment (DmitryNAleshin) [inline src/main/java/com/smartbear/mqttsupport/teststeps/panels/PublishTestStepPanel.java:98]: Any reason for this?

Comment (AlexanderYeremeyev) [reply src/main/java/com/smartbear/mqttsupport/teststeps/panels/PublishTestStepPanel.java:98]: with these changes it looks better
image

Comment (AlexanderYeremeyev) [reply src/main/java/com/smartbear/mqttsupport/teststeps/panels/PublishTestStepPanel.java:98]: image

**Review by DmitryNAleshin — APPROVED** [review]
I dont line border but let it be

**Pull request merged**